modules/ocf/manifests/motd.pp

```diff
@@ -1,12 +1,12 @@
-class ocf::motd {
-  $is_virtual = str2bool($::is_virtual)
-
-  if tagged('ocf_ssh') or tagged('ocf_apphost') {
+class ocf::motd($nfs_motd = false) {
```

Comment:
I don't think I'd name it this, since not all servers mounting NFS are using this motd, only the public-facing ones. Maybe something like `$public` or `$public_motd` would be more appropriate?